js/ui/windowManager.js

```diff
@@ -2214,21 +2226,31 @@ const CodingManager = new Lang.Class({
             return;
         if (focusedWindow === session.actorBuilder.meta_window) {
             if (session.actorApp.meta_window === previousFocused) {
-                this._rotateInActors.push(session.actorBuilder);
-                this._rotateOutActors.push(session.actorApp);
+                // make sure we do not rotate when a rotation is running
+                if (this._rotateInActors.length || this._rotateOutActors.length) {
+                    if (session.buttonBuilder)
+                        session.buttonBuilder.show();
```

Comment:
Why do we assume here that `session.buttonBuilder` may be null where below it is unconditionally used without a null check?